rfc/0000-lock-file.md

```diff
@@ -195,7 +197,15 @@ Suppose you have your own `lint-crate` inside of the workspace that needs to be
 - `Marker.lock`: hardlink to `target/marker/Cargo.lock` and contains only `marker_lints` and its `marker_api` entries
 - `target/`
     - `marker/`
-        - `Cargo.toml`: contains a definition of the `buildspace` and the "dispatch" crate. The `[dependencies]` section includes only `marker_lints = "x.y.z"`. The `path` dependencies never get into the `dependencies` here.
+        - `target/`: target directory shared between all buildspaces and all compilations including when lint crates are specified via `path`.
+        - `buildspace/`
+            - `default/`: the default buildspace that persists the `Cargo.lock` as `Marker.lock`
+                - `Cargo.toml`: contains a definition of the `buildspace` and the "dispatch" crate. The `[dependencies]` section includes only `marker_lints = "x.y.z"`. The `path` dependencies never get into the `dependencies` here.
+                - `Cargo.lock`: same as `Marker.lock`
+            - `ephemeral/`: ephemeral buildspace that doesn't persist any state outside of it. It's used when `--no-marker-lock` is specified.
```

Comment:
Bike shed:

I think `temp` would be a better name here. Mostly because this is the first time, I've heard the word *ephemeral* in this context. Either way, this shouldn't block the progress of the RFC.

Reply:
Makes sense, renamed